### PR TITLE
update setup documentation

### PR DESCRIPTION
**Pull request by tm-ger-chagas**
# Description

Setup documentation updates

## Type of change

- [x] This change requires a documentation update

### CODE REVIEW

File: assets/deploy-from-branch.png
Comment (butchtm) [inline assets/deploy-from-branch.png:1]: is this being used? I don't see a link in SETUP.md

File: assets/your-secret-here.PNG
Comment (butchtm) [inline assets/your-secret-here.PNG:1]: Is this being used? I don't see a link in SETUP.md

File: assets/workflows.png
Comment (butchtm) [inline assets/workflows.png:1]: Is this being used? I don't see a link in SETUP.md

File: assets/secrets-and-variables.PNG
Comment (butchtm) [inline assets/secrets-and-variables.PNG:1]: Is this being used? I don't see a link in SETUP.md

File: assets/run-workflow.png
Comment (butchtm) [inline assets/run-workflow.png:1]: Is this being used? I don't see a link in SETUP.md

**Review by butchtm — CHANGES_REQUESTED** [review]
Hi @tm-ger-chagas requesting some copy edits and 
maybe missing image links (or remove unused images)

File: assets/deploy-from-branch.png
Comment (tm-ger-chagas) [reply assets/deploy-from-branch.png:1]: my bad on a bunch of these, it seems that I copied an older draft that did not include the usage of these images

**Review by butchtm — APPROVED** [review]
Approved after applying some updates to split the commands for Ubuntu/MacOS

**Pull request merged**